README.adoc

```diff
@@ -274,17 +251,18 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see if the tests detect a failure, remove the change system property utility call on
-the `FaultToleranceTest.java` file. Rerun the Maven build. You see a test failure occur for the
-`testFallbackForGet()` test case.
+To see if the tests detect a failure, comment out the `changeSystemProperty()` methods
+in the `FaultToleranceTest.java` file. Rerun the Maven build.
+You see a test failure occur for the `testFallbackForGet()` test case.
+
 
 // =================================================================================================
 // Great work! You're done!
 // =================================================================================================
 
 == Great work! You're done!
 
-You have just completed building and testing a microservice application with MicroProfile Fault Tolerance and Open Liberty.
+You just learned how to build a fallback mechanism for a microservice with MicroProfile Fault Tolerance and wrote a test to validate it.
```

Comment:
Should we say, "the MicroProfile Fault Tolerance API" because we're referring to the specific named API?

Reply:
This is referring to the MicroProfile Fault Tolerance feature in OL. The API is an unimplemented version of the feature. No change.

Reply:
Yes, we should not.